### PR TITLE
Feature/popular categories

### PR DESCRIPTION
**Pull request by var-kyle**
# Description

[BF-79 Create popular category card](https://bf-decd.atlassian.net/browse/BF-79?atlOrigin=eyJpIjoiZDM5OGNkZTc5ZTJhNDUzNDlmMTg3ZTVjMzZlYzQ3MzQiLCJwIjoiaiJ9)
[BF-81 Add popular category grid to home page](https://bf-decd.atlassian.net/browse/BF-81?atlOrigin=eyJpIjoiYmIxYjk3NzUxOTUwNGE0NzliNWU3Nzc5ZWE2NTA1OTMiLCJwIjoiaiJ9)

Created a popular category card component. I created it separately from the benefit card, but they do share a lot of similarities. I kind of wanted to keep (somewhat) complex logic out of the components, but it might make more sense to create a more generic card component that benefits and categories can use.

Added the cards to the home page, but didn't put them in a grid component - for similar reasons as above, a generic component might make sense, but I just wanted these things on the page for now.

## Meets Definition of Done
- [x] Checked against style guide as currently understood
- [ ] Acceptance criteria met
- [x] Unit tests up to date
- [ ] E2E tests up to date
- [ ] Security up to standards
- [x] I18n

### CODE REVIEW

File: components/molecules/PopularCategoryCard.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import PropTypes from "prop-types";
+import { useTranslation } from "next-i18next";
+import Image from "next/image";
+
+export function PopularCategoryCard(props) {
+  const { t } = useTranslation("common");
```
Comment (DaceyTom2) [inline components/molecules/PopularCategoryCard.js:6]: translation is not being used

File: pages/index.js [hunk 2]
```diff
@@ -2,24 +2,41 @@ import Head from "next/head";
...
+  const categories = popularCatagories.map((cat) => {
+    return (
+      <PopularCategoryCard
+        key={cat.id}
+        id={cat.id}
+        title={cat.title}
+        description={cat.description}
+        type={cat.type}
+        imgSource={cat.imgSource}
+        imgAltText={cat.imgSource}
```
Comment (DaceyTom2) [inline pages/index.js:36]: imgAltText is returned from popularCatagories. Was it intended to be used here?

File: pages/index.js [hunk 3]
```diff
@@ -30,12 +47,16 @@ export default function Home({ locale }) {
         <h1 className="text-4xl text-bold">{t("findSupport")}</h1>
       </div>
 
-      <section className="">
+      <section id="popular_catagories" className="">
         <h3 className="text-2xl text-bold py-3">{t("popularCatagories")}</h3>
-        <p>{t("underConstruction")}</p>
```
Comment (DaceyTom2) [inline pages/index.js:35]: t, translate no longer serves a purpose in this file either

**Review by DaceyTom2 — CHANGES_REQUESTED** [review]

File: pages/index.js (hunk 3 above)
Comment (var-kyle) [reply pages/index.js:35]: Does it not? I see 3 other places that it's being used.

Comment (DaceyTom2) [reply pages/index.js:35]: I might be legally blind

**Review by DaceyTom2 — APPROVED** [review]
Beauty

**Pull request merged**